lib/rules/sort-comp.js

```diff
@@ -172,7 +172,7 @@ module.exports = {
         }
       }
 
-      if (method.static) {
+      if (method.static && method.name !== 'getDerivedStateFromProps') {
```

Comment:
We might want to just add `indexes.length === 0` or something like that here so all named methods would be caught here rather than just `getDerivedStateFromProps` for future extensibility.

Reply:
Thanks, checking for `indexes.length === 0` will do the job. Fixed